scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -55,6 +55,18 @@
   }
 }
 
+.nav-pills {
+  margin-bottom: 15px;
+
+  > li {
+    // Active state
+    &.active > a {
```

Comment:
why only for the active ones? i see the problem with `:focus` also on the non-active items